_includes/page-header.html

```diff
@@ -3,7 +3,7 @@ <h1 class="project-name">{{ site.title }}</h1>
   <h2 class="project-tagline">{{ site.tagline }}</h2>
   <a href="/" class="btn">About</a>
   <!-- <a href="/process" class="btn">DALI Process</a> -->
-  <a href="/milestones" class="btn">Milestones</a>
+  <a href="/milestones" class="btn">Workflow</a>
```

Comment:
I like this change - i see you chose not to break out entire link structure as well